### DISCUSSION TITLE
Welcome to ternoa-proposals Discussions!

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/ternoa-proposals#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * **Share ideas**.
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Balianos** [comment]
Alias / Location
Julian SCHMERKIN aka Tchiki 36 yo living in Paris 🇫🇷

Background / Experience
Community creator/grower in Gaming & Web3 (RDR2 Online FR, NBA2kFr, Gaming2kFr, AmisCrypto, Thugz Life NFT project), Freelance UX/UI Integrator & Dev Front JS, and a Community Manager / Account Manager /Project Manager experience.

How I'd like to get involved in Ternoa :
By giving the best of my knowledges and experience to bring new ideas for Ternoa & Capsule Corp innovations bout NFTs

Areas of interest
Rap/HipHop, Gaming, Basketball/Soccer, Dev, 3D (new), Old Rhum, Politic, Geek Culture, Movies, TV shows...

What about Gaming... ?
Old ProGamer on Counter-Strike (Beta 1.5, 1.6 with SAS Clan), NBA 2k, Football Manager, PES/FIFA, GTA, UFC, Madden, Rocket League, a lot...

What about Web3... ?
I explored a lot of domains in Web3 world : Crypto Investment, Crypto Trading with Technical Analysis learning, DeFi (on Ethereum, BSC, Solana, AVAX), Smart Contracts coding POC on Solidity & Achetype/Michelson, NFTs and Metaverse.
I got more experience on the last 2 domains : NFT & Metaverse. 
I founded one year ago the [Thugz Life](https://thugz.life) project and I'm managing it in almost everything apart of the visual creation.
Fun fact : we organized the first 100% French rap showcase in the Metaverse
